src/bartholomew.rs

```diff
@@ -98,17 +80,36 @@ pub fn render(req: Request) -> Result<Response> {
     engine.load_script_dir()?;
 
     // Load the content.
-    let content_path = match dynamic_template_params {
-        Some(val) => content::content_path(PathBuf::from(CONTENT_PATH), &val.dynamic_content_path),
-        None => content::content_path(PathBuf::from(CONTENT_PATH), &path_info),
-    };
+    let content_path = content::content_path(PathBuf::from(CONTENT_PATH), &path_info);
 
     eprintln!("Path {}", content_path.to_string_lossy());
 
     match std::fs::read_to_string(&content_path) {
         Ok(full_document) => {
             let mut doc: content::Content = full_document.parse()?;
 
+            // If a dynamic content source if specified get the body from that file instead
+            if let Some(val) = &doc.head.dynamic_body_source {
+                match std::fs::read_to_string(content::content_path(
+                    PathBuf::from(CONTENT_PATH),
+                    val,
+                )) {
+                    Ok(alternate_content) => {
+                        let new_content: content::Content = alternate_content.parse()?;
+                        doc.body = new_content.body;
+                    }
+                    Err(_) => {
+                        let err_vals = template::error_values(
+                            "Not Found",
+                            "The alternate content source specified on page was not found.",
```

Comment:
This shouldn't be presented to users - nobody reading the site needs to know about alternate content sources.  Log the error instead (and, I would suggest, capture the actual I/O error rather than assuming it must be 'file not found').